Storage/Rust/RustShared.swift

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+private let log = Logger.syncLogger
+
+public class RustShared {
```

Comment:
Was this accidental? This file shouldn't be added here otherwise we're gonna collide when we merge the rust-bookmarks branch.